src/Type/Comment/Connection/CommentConnectionArgs.php

```diff
@@ -189,9 +190,8 @@ private static function fields() {
 	 */
 	private static function comments_orderby_enum() {
 
-		if ( null === self::$comments_orderby_enum ) {
-
-			self::$comments_orderby_enum = new EnumType([
+		if ( null === self::$comments_orderby_enum ) :
+			self::$comments_orderby_enum = new WPEnumType([
```

Comment:
Are we using just `EnumType` other places? Should probably get an issue going to convert all `EnumType`'s to `WPEnumType`